DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -92,6 +68,66 @@ void pat::PackedCandidate::unpack() const {
       p4c.release();
     }
 }
+
+/*void pat::PackedCandidate::legacyCovarianceUnpacking() const {
```

Comment:
this will be removed once we verify that the backward compaibility as currently implemented works